platform-sdk/swirlds-virtualmap/src/test/java/com/swirlds/virtualmap/VirtualMapHashingTest.java

```diff
@@ -430,9 +428,4 @@ void fullLeavesRehashOnEmptyMap() {
         // shouldn't throw any exceptions
         assertDoesNotThrow(() -> doFullRehash(root));
     }
-
-    @AfterEach
-    void tearDown() {
-        MerkleDbTestUtils.assertAllDatabasesClosed();
```

Comment:
Tests in `swirlds-virtualmap` don't use MerkleDb, no need to have this check, it just adds a module dependency for no good use